src/main/java/nextstep/courses/service/EnrollSessionService.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.courses.service;
+
+import nextstep.courses.domain.Student;
+import nextstep.courses.domain.session.RegistrationState;
+import nextstep.courses.domain.session.Session;
+import nextstep.courses.domain.session.SessionType;
+import nextstep.users.domain.NsUser;
+
+public class EnrollSessionService {
+    public Student enrollSession(Session session, NsUser student) {
+        Student studentInfo = new Student(student.getId(), session.getId(), RegistrationState.PENDING);
+        if (SessionType.isFree(session.getSessionType())) {
+            session.signUp(studentInfo);
+        } else {
+            session.signUp(studentInfo);
+        }
+        return studentInfo;
+    }
```

Comment:
```suggestion
    public Student enrollSession(Session session, NsUser student) {
        return session.signUp(student);
    }
```
위와 같이 구현하는 것은 어떨까?
위와 같이 구현한다면 굳이 이 메서드가 필요할까?